slides/developer-guide/working-with-images/get-image-information-from-a-powerpoint-document/_index.md

```diff
@@ -6,518 +6,334 @@ weight: 10
-|**API**|**Type**|**Description**|**Swagger Link**|
-| :- | :- | :- | :- |
-|/slides/{name}/images|GET|Get all image information from a Power Point Document|[GetSlidesImages](https://apireference.aspose.cloud/slides/#/Images/GetSlidesImages)|
-### **cURL Example**
-{{< tabs tabTotal="2" tabID="1" tabName1="Request" tabName2="Response" >}}
-
-{{< tab tabNum="1" >}}
-
-**Create Request Token**
-
-```java
-
-curl -v "https://api.aspose.cloud/connect/token" -X POST -d "grant_type=client_credentials&client_id=78946fb4-3bd4-4d3e-b309-f9e2ff9ac6f9&client_secret=b125f13bf6b76ed81ee990142d841195" -H "Content-Type: application/x-www-form-urlencoded" -H "Accept: application/json"
-
-```
-
-```java
-
-curl  -v -X GET "https://api.aspose.cloud/v3.0/slides/presentation_images.pptx/images" -H "Content-Type: application/json" -H "Authorization: Bearer eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJuYmYiOjE1NTk4NDY0NTQsImV4cCI6MTU1OTkzMjg1NCwiaXNzIjoiaHR0cHM6Ly9hcGkuYXNwb3NlLmNsb3VkIiwiYXVkIjpbImh0dHBzOi8vYXBpLmFzcG9zZS5jbG91ZC9yZXNvdXJjZXMiLCJhcGkucGxhdGZvcm0iLCJhcGkucHJvZHVjdHMiXSwiY2xpZW50X2lkIjoiNzg5NDZmYjQtM2JkNC00ZDNlLWIzMDktZjllMmZmOWFjNmY5Iiwic2NvcGUiOlsiYXBpLnBsYXRmb3JtIiwiYXBpLnByb2R1Y3RzIl19.jeWJiVCVWeRJ_1Uux9PP701XZ7inSf2g1gotFTBkb76tWBOJf7cASswVUM7gXhMKSZ7suYwRl_ZC0KmGHUiVjCBQZrjsAAAvwZsFIHLZMd_H-hbZti4XPjRSChUYzTF1kJ1vnvLEABXPf4yybZqNdbqe5zBMtsWuVFtaNs_V7PkVhz_e3v1L6nYKw84VbINMOCwqMXd2EwRilzt7VsKWBL47A1AY1D3b-Mp4xrKoc5ICWUAA_qqEOaPmp1V8GF0OzjFgn6FKeeSa13pIbD7Xt6qVZDpK2mm11vGnlGE3FMgSUz7HWempfsNyo1KpAiDhQG4VS6wl3QYdyVB7EzwyPA" --ssl-no-revoke
-
-```
-
-{{< /tab >}}
-
-{{< tab tabNum="2" >}}
-
-```java
-
-{
-
-   "list":[
-
-      {
-
-         "width":275,
-
-         "height":183,
-
-         "contentType":"image/jpeg",
-
-         "selfUri":{
-
-            "href":"https://api.aspose.cloud/v3.0/slides/presentation_images.pptx/images/1",
-
-            "relation":"self"
-
-         },
-
-         "alternateLinks":[
-
-            {
-
-               "href":"https://api.aspose.cloud/v3.0/slides/presentation_images.pptx/images/1/jpeg",
-
-               "relation":"alternate"
-
-            },
-
-            {
-
-               "href":"https://api.aspose.cloud/v3.0/slides/presentation_images.pptx/images/1/png",
-
-               "relation":"alternate"
-
-            },
-
-            {
-
-               "href":"https://api.aspose.cloud/v3.0/slides/presentation_images.pptx/images/1/gif",
-
-               "relation":"alternate"
-
-            }
-
-         ]
-
-      },
-
-      {
-
-         "width":299,
-
-         "height":168,
-
-         "contentType":"image/jpeg",
-
-         "selfUri":{
-
-            "href":"https://api.aspose.cloud/v3.0/slides/presentation_images.pptx/images/2",
-
-            "relation":"self"
-
-         },
 
-         "alternateLinks":[
+The following request method allows you to easily read information about all images contained in a PowerPoint document. You can find out the following image properties: width, height, type, etc.

```

Comment:
The following method allows you to easily read information about all images contained in a PowerPoint document. You can get such image properties as width, height, type, etc.